### PR TITLE
Fixed bzr-2.6.0 package

### PR DESCRIPTION
**Pull request by yaso195**
Earlier, it was not being installed properly. Any app that has been created
from this package were not having the bzr package. Now, it is fixed.

@zquestz @variadico

### CODE REVIEW

File: packages/bzr-2.6.0.conf [hunk 1]
```diff
@@ -7,24 +7,23 @@ sources [
...
 
-environment { "PATH": "/opt/apcera/bzr-2.6/bin:$PATH" }
+environment { "PATH": "/opt/apcera/bzr-2.6.0/bin:$PATH" }
 
 # The python package provides the needed ENV variables to build this package.
 # This includes PYTHONHOME, PYTHONPATH and C_INCLUDE_PATH.
 build (
-      export INSTALLPATH=/opt/apcera/bzr-2.6.0
-      export PREFIX=${INSTALLPATH}
+export INSTALLPATH=/opt/apcera
```
Comment (zquestz) [inline packages/bzr-2.6.0.conf]: Why the whitespace removal?

**Comment by zquestz** [comment]
LGTM other than nit.

File: packages/bzr-2.6.0.conf [hunk 2]
```diff
@@ -7,24 +7,23 @@ sources [
     sha2: "5eb4d0367c6d83396250165da5bb2c8a9f378293" },
 ]
 
-build_depends [ { package: "build-essential" },
-                { runtime: "python-2.7" } ]
```
Comment (tw4dl) [inline packages/bzr-2.6.0.conf]: Does it have to be python-2.7? How about just "python" if it works with both python 2 or 3

File: packages/bzr-2.6.0.conf [hunk 3]
```diff
@@ -7,24 +7,23 @@ sources [
...
-depends  [ { os: "ubuntu" } ]
+build_depends [ { package: "build-essential" } ]
+depends  [ { os: "ubuntu" },
+           { runtime: "python-2.7" } ]
```
Comment (tw4dl) [inline packages/bzr-2.6.0.conf]: same here.

File: packages/bzr-2.6.0.conf (hunk 2 above)
Comment (yaso195) [inline packages/bzr-2.6.0.conf]: the build fails, when only `python` is given

**Comment by yaso195** [comment]
Fixed the nit

**Pull request merged**